django_project/api/osm_api_client.py

```diff
@@ -0,0 +1,263 @@
+# -*- coding: utf-8 -*-
+
+import time
+
+from osmapi import (
+    OsmApi, OsmApiError, ApiError,
+    ResponseEmptyApiError, ElementDeletedApiError)
+from requests_oauthlib import OAuth1
+
+
+class OAuthTokenMissingError(OsmApiError):
+    """
+    Error when oauth token is missing for an authenticated request
+    """
+    pass
+
+
+class OsmApiWrapper(OsmApi, object):
+
+    def __init__(
+            self,
+            client_key,
+            client_secret,
+            oauth_token,
+            oauth_token_secret,
+            api,
+            appid):
```

Comment:
i think we could remove this, and move app_id into django setting
and this class can use the setting

Reply:
my intention was to make this wrapper client generic so it could be used by any other app. For example when I create unittest for this particular class, I set the app id to `healthsites.testcase`. Is it ok?

Reply:
ok @myarjunar 
that is nice idea
maybe you could use default value?
`appid=settings.OSMAPPID?`

so when in production/development (not in test), the code keep using settings (withouth